app/e2e/client/rocketchat.e2e.js

```diff
@@ -174,10 +154,30 @@ class E2E {
 							return;
 						}
 						localStorage.removeItem('e2e.randomPassword');
+						call('e2e.updateSavedKeyUser');
 						this.closeAlert();
 					});
 				},
 			});
+		}	else if (!private_key && this.db_private_key) {
+			try {
+				private_key = await this.decodePrivateKey(this.db_private_key);
+			} catch (error) {
+				this.started = false;
+				failedToDecodeKey = true;
+				this.openAlert({
+					title: TAPi18n.__('Wasn\'t possible to decode your encryption key to be imported.'),
+					html: '<div>Your encryption password seems wrong. Click here to try again.</div>',
```

Comment:
Both `title` and `html` properties need newest i18n strings added to at least `en.i18n.json`.

Reply:
ok, will add those